### ISSUE TITLE
Removed items being passed on POST

### ISSUE THREAD

**Issue by orthodoxws** [arendjr/selectivity#196]
Items selected, then unselected are still being passed with the form data:

https://www.screencast.com/t/oVtooqGwl3k

**Comment by arendjr** [comment]
Sorry, I don't have Flash installed here, can you describe the issue in some other way?

**Comment by orthodoxws** [comment]
Ah, I'm sorry!  Here it is as an MP4:  https://www.screencast.com/t/feWNGeWir

**Comment by arendjr** [comment]
Thanks! I suppose you're using the jQuery build, right? Can you show me the code you use for initializing Selectivity? Am I understanding correctly it is only removing that's causing an issue, right?

**Comment by orthodoxws** [comment]
Yes, I'm using the jQuery build.  The problem is that when I remove a directory which I had formerly checked, it still submits in the POST data.  Thanks for your help with this!

Here's the code:

$(".selectivity").selectivity( { backspaceHighlightsBeforeDelete: true } );

Then..

														<select id="cb" name="cb[]" multiple class="selectivity">
															<?
															
														  for ($s = 0; $s < $snum_results; $s++)
														  {
														     echo "<option value=\"$srow[dir]\" ";
														     if (in_array($srow[dir], $dir_arr))
														         echo "selected";
														     echo "> $srow[dir]</option>";
														     $srow = mysqli_fetch_array($sresult);
														  }															
																																																																 
															?>   
														</select>

**Comment by arendjr** [comment]
Thanks for that code sample! I think what is going wrong is that the
selected options you generate in your markup are still being sent when you
submit the form. I can agree that's a bug that should be handled by
Selectivity, but I have to think how I can easily solve that. Will keep you
posted!

**Comment by orthodoxws** [comment]
Thank you very much! I appreciate it!
